server/events_controller.go

```diff
@@ -57,6 +57,9 @@ type EventsController struct {
 	// request validation is done.
 	GitlabWebhookSecret  []byte
 	RepoWhitelistChecker *events.RepoWhitelistChecker
+	// Signifies if we post a comment when an event comes in for a non-whitelisted
```

Comment:
Comment needs to start with `// SilenceWhitelistErrors` as per golang comment style guide:

> Notice this comment is a complete sentence that **begins with the name of the element it describes**... 
> https://blog.golang.org/godoc-documenting-go-code

ex:
```
// SilenceWhitelistErrors controls whether we write an error comment on pull requests from non-whitelisted repos.
```